sdk/storage/Azure.Storage.Common/src/Shared/StorageResponseClassifier.cs

```diff
@@ -50,7 +50,12 @@ public override bool IsErrorResponse(HttpMessage message)
             switch (message.Response.Status)
             {
                 case 409:
```

Comment:
Should we do 412 here as well?  We've had requests around that as well, since we expect preconditions to fail during concurrent operations; it also isn't an exceptional situation.

Reply:
Looking at https://docs.microsoft.com/en-us/rest/api/storageservices/blob-service-error-codes I don't see anything I want to eagerly add for now.